lib/chef/knife/openstack_floating_ip_delete.rb

```diff
@@ -0,0 +1,26 @@
+#
+# Author:: Vasundhara Jagdale (<vasundhara.jagdale@clogeny.com>)
+# Copyright:: Copyright (c) 2013 Chef Software, Inc.
+#
+\
+require 'chef/knife/openstack_helpers'
+require 'chef/knife/cloud/openstack_service_options'
+
+class Chef
+  class Knife
+    class Cloud
+      class OpenstackFloatingIpDelete < Command
+        include OpenstackServiceOptions
+        include OpenstackHelpers
+
+        banner 'knife openstack floating_ip delete id [ID] (options)'
```

Comment:
Instead of just `delete` it should have an option for `deallocate` or `release`, if possible. The OpenStack terminology is to `deallocate` or `release` for getting releasing IPs.